### PR TITLE
Devops/gitpod

### PR DESCRIPTION
**Pull request by codecakes**
This change is [Reviewable](https://reviewable.io/reviews/xcov19/covidx/100)

### CODE REVIEW

File: .gitpod.yml [hunk 1]
```diff
@@ -1,8 +1,8 @@
 image:
   file: Dockerfile
 tasks:
-  - init: touch .env
-    command: DEBUG_ENV=1 SECRET_KEY=dskaj343 CPPFLAGS="$(pg_config --cppflags)" LDFLAGS="$(pg_config --ldflags)" bazel build :manage --watchfs --spawn_strategy=standalone --copt --aspects=@bazel_tools//tools/python:srcs_version.bzl%find_requirements --verbose_failures=true --show_timestamps=true --python_version=PY3 --build_python_zip --sandbox_debug --color=yes --curses=yes --jobs=2000 --loading_phase_threads=HOST_CPUS --action_env=LDFLAGS="$(pg_config --ldflags)" --action_env=CPPFLAGS="$(pg_config --cppflags)" --action_env=DEBUG_ENV --action_env=SECRET_KEY
+  - init: sudo docker-up
```
Comment (marcelloromani) [inline .gitpod.yml:4]: What is `docker-up` ?

Comment (codecakes) [reply .gitpod.yml:4]: preview feature on gitpod that enables docker compose to run in root

**Comment by codecakes** [comment]
Retry DeepCode

**Review by marcelloromani — APPROVED** [review]
`docker-compose up --build` OK
http://localhost:8000/apihealth OK
so I guess it's OK

Haven't tested the gitpod functionality itself as I don't know how to do that.

**Pull request merged**